### PR TITLE
20221205 caching

### PR DESCRIPTION
**Pull request by Chiuzu-Chilumbu**

### CODE REVIEW

**Comment by Chiuzu-Chilumbu** [comment]
@drainvers @javaku9527 Have a look before I deploy to Heroku.

File: app/presentation/representers/trip_representer.rb [hunk 1]
```diff
@@ -16,8 +16,8 @@ class Trip < Roar::Decorator
       property :id
       property :query_id
       property :currency, extend: Representer::Currency, class: OpenStruct
-      property :origin                   # TODO: extend: Representer::Airport, class: OpenStruct
-      property :destination              # TODO: extend: Representer::Airport, class: OpenStruct
+      property :origin, extend: Representer::Airport, class: OpenStruct
+      property :destination, extend: Representer::Airport, class: OpenStruct
```
Comment (javaku9527) [inline app/presentation/representers/trip_representer.rb:20]: Add the airport representer in ComfyWings::Representer::Flight origin and destination

File: config/environment.rb [hunk 2]
```diff
@@ -7,38 +7,68 @@
...
 require 'delegate'
+require 'rack/cache'
+require 'redis-rack-cache'
 
 module ComfyWings
   # Configuration for the App
   class App < Roda
     plugin :environments
 
-    # rubocop:disable Lint/ConstantDefinitionInBlock
```
Comment (javaku9527) [inline config/environment.rb:16]: Too many duplicate code, I think the merge conflict is failed

File: _cache/rack/body/69/6d474617a4e0d4abbc44db81b077efc586e2ae [hunk 3]
```diff
@@ -0,0 +1 @@
+{"currencies":[{"code":"TWD","name":"New Taiwan dollar"},{"code":"USD","name":"United States dollar"},{"code":"EUR","name":"Euro"},{"code":"GBP","name":"Pound sterling"}]}
```
Comment (javaku9527) [inline _cache/rack/body/69/6d474617a4e0d4abbc44db81b077efc586e2ae:1]: the file under directory `_cache` shouldn't push to git repository. add _cache/* into .gitignore

**Review by javaku9527 — COMMENTED** [review]
@Chiuzu-chilumbu found something need to fix. you may merge when you finish these problem

**Pull request closed without merging**